newdle/client/src/components/answer/AnswerPage.js

```diff
@@ -55,6 +59,16 @@ export default function AnswerPage() {
     setSaved(true);
   }
 
+  useEffect(() => {
+    if (newdle && participantCode) {
+      dispatch(fetchParticipant(newdle.code, participantCode));
+    }
+  }, [newdle, participantCode, dispatch]);
+
+  useEffect(() => {
+    dispatch(fetchBusyTimesForAnswer(participantEmail, dates));
```

Comment:
this breaks if you do not have a participant, doesn't it?

Reply:
It didn't break for some reason but fixed :)